js/templates/transactions/table/row.html

```diff
@@ -0,0 +1,116 @@
+<%
+  // Pass in a type of 'sales', 'purchases' or 'cases'
+%>
+
+<% var type = ob.type || 'sales' %>
+
+<%
+  var types = ['sales', 'purchases', 'cases'];
+
+  if (types.indexOf(ob.type) === -1) {
+    throw new Error('The type must be one of ${types}');
+  }
+
+  if (!ob.userCurrency) {
+    throw new Error('Please provide the users local currency.');
+  }
+
+  if (!ob.moment) {
+    throw new Error('Please pass in the moment library.');
+  }
+%>
+
+<td class="clrBr orderCol noOverflow">
+  <div class="unreadBorder clrE1"></div>
+  <%= ob.orderId %>
+</td>
+<td class="clrBr dateCol"><%= ob.moment(ob.timestamp).format('l LT') %></td>
+<td class="clrBr listingCol">
+  <div class="flexVCent gutterHSm">
+    <% const imageHashes = { small: ob.thumbnail, tiny: ob.thumbnail } %>
```

Comment:
This is a little ugly, but IMO getListingBgImage needs to be extracted to be used for different listing image sizes. Maybe it would accept a target width and height and then it would select the appropriate image hash based off of that as opposed to what it's doing now, which is flat out using small for hiRez and tiny otherwise.

It would mean the responsive module would need to know what the different sizes are (e.g. small is X by X, whereas tiny is X by X, etc...), but I think that's ok.

In any case, I think this dirty (super easy) approach I used, for now, is probably ok.